src/encoded/static/components/objectutils.js

```diff
@@ -634,3 +634,28 @@ AlternateAccession.propTypes = {
 AlternateAccession.defaultProps = {
     altAcc: null,
 };
+
+
+/**
+ * Display a list of internal_tags for an object as badges that link to a corresponding search. The
+ * object in `context` must have at least one `internal_tags` value or the results are
+ * unpredictable.
+ */
+export const InternalTags = ({ context, css }) => {
+    const tagBadges = context.internal_tags.map((tag) => {
+        const tagSearchUrl = `/search/?type=${context['@type'][0]}&internal_tags=${globals.encodedURIComponent(tag)}`;
+        return <a href={tagSearchUrl} key={tag}><img src={`/static/img/tag-${tag}.png`} alt={`Search for all ${context['@type'][0]} with internal tag ${tag}`} /></a>;
+    });
+    return <span className={css}>{tagBadges}</span>;
+};
+
+InternalTags.propTypes = {
+    /** encode object being displayed */
+    context: PropTypes.object.isRequired,
+    /** Optional CSS class to assign to <span> surrounding all the badges */
+    css: PropTypes.string,
```

Comment:
It doesn't look like you're using "css" on any of the pages where you're including this component. You think it might be useful in the future, though?

Reply:
Almost certainly I used the `css` prop in the past. I figured it could be useful in the future.